models/model.js

```diff
@@ -37,6 +37,17 @@ exports.fetchUsers = () => {
 
 exports.fetchArticles = () => {
     return db.query("SELECT articles.author, articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, CAST(COUNT(comments.article_id) AS INT) AS comment_count FROM articles LEFT JOIN comments ON articles.article_id = comments.article_id GROUP BY articles.article_id ORDER BY articles.created_at DESC")
-
     .then((result) => result.rows)
+}
+
+exports.fetchArticleComments = (article_id) => {
+    if (/[^\d]/gi.test(article_id)) {
+        return Promise.reject(({status: 400, msg: "Invalid article_id"}))
+    }
+    return db.query("SELECT * FROM comments WHERE article_id = $1", [article_id])
+    .then((result) => {
+        if (result.rows[0] === undefined){
+            return Promise.reject({status: 404, msg: "Article not found"})
+        } else return result.rows;
```

Comment:
You are going to be sending back an empty array for an Article that exists but has no comments. This I believe is the correct behaviour, I just wanted to make sure you are aware